buildSrc/src/main/kotlin/ktx/Versions.kt

```diff
@@ -7,7 +7,7 @@ const val artemisOdbVersion = "2.3.0"
 const val ashleyVersion = "1.7.4"
 const val gdxAiVersion = "1.8.2"
 const val visUiVersion = "1.5.5"
-
```

Comment:
I think that line might have been deliberate to separate test dependency versions. It's a tiny thing, but if you don't mind putting it back I'd appreciate it. :smile: